hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -153,6 +154,7 @@ data HeadState tx
         readyToFanoutSent :: Bool
       , chainState :: ChainStateType tx
       , headId :: HeadId
+      , closedContesters :: [Plutus.PubKeyHash]
```

Comment:
Must: not use cardano-specific types in `HeadLogic`.

Why do we even track it in the `HeadLogic`? I could see value in having this information, but I do want to avoid chain-specific things here AT ALL COSTS. So we would need to define a `type family` on `tx` for this to keep it abstract. But then.. is it worth it without a clear user story?

I would say, let's not track the list of contesters at the logic-level for now.